src/components/Calculator.js

```diff
@@ -16,6 +16,9 @@ const Calculator = () => {
 
   return (
     <div className="App">
+      <div className="Heading-2">
+        <h2> Let&#39;s do some math!</h2>
+      </div>
       <div className="calculator">
         <div className="display">
           {state.total}
```

Comment:
[REQUIRED] Kindly align the text to the left
![image](https://user-images.githubusercontent.com/62938735/213865446-0ddf5238-3281-4766-aee7-5501ce3841dd.png)